src/test/java/uk/ac/ebi/subs/filecontentvalidator/validators/VcfFileValidatorTest.java

```diff
@@ -74,7 +76,7 @@ public void test_file_finding() throws IOException {
 
     @Test
     public void test_outputDir_is_a_directory() throws IOException {
```

Comment:
should this test move out of VcfFileValidatorTest, as it's now testing a method in a different class?

Reply:
I agree. I can move this and test_file_finding method to a separate test class